src/routes/Forms.jsx

```diff
@@ -42,26 +39,14 @@ const Forms = () => {
         justifyContent: 'center'
       }}
     >
-      <Button
-        variant="contained"
-        color="secondary"
-        size="small"
-        aria-label="Clear Client Button"
-        startIcon={<Remove />}
-        onClick={() => setSelectedUser('')}
-        sx={{ margin: '2rem 0 1rem' }}
-      >
-        Clear Client
-      </Button>
-      <Typography>Client selected: {selectedUser.person || 'No client selected'}</Typography>
+      <Typography>Client selected: {selectedUser.person || selectedUser.podUrl}</Typography>
```

Comment:
Mentioned about setting default selectedUser.person as "Personal Pod". Open to other options to indicate the user's own Pod.

Reply:
fixed